synapse/rest/key/v2/remote_key_resource.py

```diff
@@ -213,7 +215,7 @@ async def query_keys(self, request, query, query_remote_on_cache_miss=False):
         else:
             signed_keys = []
             for key_json in json_results:
-                key_json = json.loads(key_json)
+                key_json = json.loads(key_json.decode("utf-8"))
```

Comment:
Maybe this should be using `db_to_json` instead? Feels really weird to put that into the REST layer though.

Reply:
well, arguably `get_server_keys_json` should be manipulating the results so as not to return `memoryview` objects.

I think this is fine though.

Reply:
I suspect it is doing it for efficiency since only some  of the results actually end up being deserialized into JSON objects? Not sure though.